experimental/federated_plugin/README.md

```diff
@@ -0,0 +1,20 @@
+# federated_plugin
+
+A Flutter plugin sample that shows how to implement federated plugin to fetch device location on different platforms.
```

Comment:
nit: "implement a federated plugin to fetch the device location on different platforms."